Makefile

```diff
@@ -10,8 +16,10 @@ fmt:
 vet:
 	go vet ./...
 
+.SILENT: test
+.PHONY: test
 test: fmt vet
-	go test ./...
```

Comment:
I think it would be good if we split the recipe into test and testacc so that the quick tests can be run easily.

Reply:
As discussed, we'll wait with this until it becomes an issue.